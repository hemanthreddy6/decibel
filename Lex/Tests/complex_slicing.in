audio music <- load "music.wav";
audio trimmed <- music[(20 / 5) : (4 * 10)][10: (2 * 10)];
save trimmed -> "trimmed.wav";