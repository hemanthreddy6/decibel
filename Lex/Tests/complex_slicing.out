AUDIO
IDENTIFIER
<-
LOAD
STRING_LITERAL
;
AUDIO
IDENTIFIER
<-
IDENTIFIER
[
(
INT_LITERAL
/
INT_LITERAL
)
:
(
INT_LITERAL
*
INT_LITERAL
)
]
[
INT_LITERAL
:
(
INT_LITERAL
*
INT_LITERAL
)
]
;
SAVE
IDENTIFIER
->
STRING_LITERAL
;
